package.json

```diff
@@ -21,14 +21,12 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "ember-cli-babel": "^6.0.0",
-    "ember-cli-version-checker": "^2.0.0",
     "ember-compatibility-helpers": "^0.1.0",
     "ember-macro-helpers": "^0.16.0"
   },
   "devDependencies": {
+    "@ember-decorators/babel-transforms": "^0.0.0",
```

Comment:
I think this should be a dependency not a devDependency. This way it will be auto installed into a consumer app

Reply:
Unfortunately, not exactly. If we list it as a `dependency`, this only means that `babel-transforms` will handle transpilation for `ember-decorators`, but not the consumer app / addon.

Reply:
I go into detail about this in the README of `babel-transforms`. 😊

Reply:
I do think @ro0gr is correct though, since we are importing `@ember-decorators/babel-transforms` in `index.js` we are using it now, just not for transforms

Reply:
Oh, absolutely correct, now that we actually import it. :sweat_smile: